__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../db/connection.js')
+const request = require('supertest')
+const seed = require('../db/seeds/seed.js')
+const data = require('../db/data/test-data/index.js')
+const app = require('../app.js')
+
+beforeEach(() => seed(data));
+afterAll(() => db.end())
+
+describe('/api/topics', () => {
+    test('GET 200: should return an array of all topics objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            const rows = response.body.rows
+            expect(rows).toHaveLength(3)
```

Comment:
Nice to see an assertion for the length, just to ensure the array contains some elements.